docs/user-guide.md

```diff
@@ -43,6 +43,7 @@ Usage of kube-router:
       --bgp-graceful-restart                          Enables the BGP Graceful Restart capability so that routes are preserved on unexpected restarts
       --bgp-graceful-restart-deferral-time duration   BGP Graceful restart deferral time according to RFC4724 4.1, maximum 18h. (default 6m0s)
       --bgp-graceful-restart-time duration            BGP Graceful restart time according to RFC4724 3, maximum 4095s. (default 1m30s)
+      --bgp-holdtime float                            This parameter is mainly used to modify the holdtime declared to BGP peer. When Kube-router goes down abnormally, the local saving time of BGP route will be affected.Holdtime must be in the range 3 to 65536. (default 90)
```

Comment:
@cloudnativer I'm ready to merge this, I just realized something really quickly. Every other unit of time in kube-router is specified via duration which is more idiomatic for Golang. Would you mind converting this to be a unit of duration like the other options?

Very sorry I missed this on my first review pass, but everything else looks good to go, even had a chance to test it in a live cluster, so once this change is made I'll merge.

Reply:
ok.
I have been busy with my work recently, so I didn't reply in time. 
I will solve this problem in three days.

Reply:
@aauren 
I have changed the type of BGPHoldtime to duration. Please check it again.